op/model.py

```diff
@@ -71,15 +71,42 @@ def get(self, entity_type, *args):
             self._weakrefs[key] = entity
         return entity
 
-
 class Application:
```

Comment:
One line too many removed above. We still want to keep the 2 empty lines between types for the general case (unless there's a better reason to pack them together such as in the case of very small types).